lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -299,7 +299,7 @@ catch throttling exceptions to work correctly), you'd need to provide a backoff
 and then put exception handling around the backoff function.
 
 You can use `exponential_backoff` or `jittered_backoff` strategies - see
-the [cloud module_utils](/tree/devel/lib/ansible/module_utils/cloud.py)
+the [cloud module_utils](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/cloud.py)
```

Comment:
Looking at https://blog.github.com/2013-01-31-relative-links-in-markup-files/ this should just be `lib/ansible/module_utils/cloud.py`